clicks.view.lkml

```diff
@@ -531,26 +590,35 @@ view: clicks {
     group_label: "Counts"
   }
 
+  # max_click_index
```

Comment:
Should this be hidden where it is an index.

Reply:
The fields `max_click_index` and `average_clicks_per_visit` use `click_in_session_index` to calculate their values. These are useful for single-number Looks answering questions like "how many clicks to reach this page on average?" Which I think are important analytics to have available.

I can see your point with hiding `click_in_session_index`, since it is more complex.